src/components/30-organisms/table-sortable/demo.js

```diff
@@ -7,44 +7,58 @@ const model = {
   thead: [
     { html: 'Age', sort: 'ASC' },
     { html: 'Name', sort: 'ASC' },
-    { html: 'City', sort: 'DESC' },
+    { html: 'City', sort: 'ASC' },
+    { html: 'Date of entry', sort: 'DESC' },
+    { html: 'Date of exit', sort: 'DESC' },
     { html: 'Department' },
   ],
   tbody: [
     [
       { html: '<span>55</span>' },
       { html: '<span>Peter</span>' },
       { html: '<span>Winterthur</span>' },
+      { html: '<span>22.04.2019</span>' },
+      { html: '<span>10.01.2020</span>' },
```

Comment:
should we add also a differrent format for test reasons? like `dd/mm/yyyy` and `dd-mm-yyyy`? We can also mix them up

Reply:
I also thought about it. I'll us the other formats in the demo.

Reply:
done